packages/x-charts/src/ChartsTooltip/ChartsTooltipContainer.tsx

```diff
@@ -187,13 +233,13 @@ function ChartsTooltipContainer(inProps: ChartsTooltipContainerProps) {
     <NoSsr>
       {isOpen && (
         <ChartsTooltipRoot
+          {...other}
           className={classes?.root}
           open={isOpen}
-          placement={isMouse ? 'right-start' : 'top'}
+          placement={other.placement ?? position ?? (isMouse ? 'right-start' : 'top')}
           popperRef={popperRef}
           anchorEl={anchorEl}
```

Comment:
Popper takes an `anchorEl` and we're calculating the coordinates to mimic the anchor element. When using `computedAnchor === 'node'`, could we not use `anchorEl` directly instead of having to implement the anchoring logic ourselves?